brainiak/utils/fmrisim.py

```diff
@@ -443,11 +447,9 @@ def generate_stimfunction(onsets,
     if len(weights) == 1:
         weights = weights * len(onsets)
 
-    # How many elements per second are you modeling
-    temporal_resolution = 1000
-
-    # Generate the time course as empty, each element is a millisecond
-    stimfunction = [0] * round(total_time * temporal_resolution)
+    # Generate the time course as empty, each element is a millisecond by
+    # default
+    stimfunction = [0] * round(int(total_time * temporal_resolution))
```

Comment:
@CameronTEllis, for some reason my previous comment (https://github.com/IntelPNI/brainiak/pull/189#discussion_r103521681) does not show up on Slack, so I am mentioning you to make sure you see it.